_faq/bugs-play-downloaded-videos.html

```diff
@@ -0,0 +1,17 @@
+---
+categories:
+- player
+- bugs
+type: info
+title: "Play downloaded videos in-app"
+---
+<section>
+    <p>In order to play downloaded videos, you need to use an external video player.</p>
+    <p>There have been considerations implementing this directly into NewPipe. This idea was dropped due to these reasons:</p>
+    <ul>
+        <li>A phone is usually shipped with a video player.</li>
+        <li>NewPipe is not designed to be an all-around video player.</li>
+        <li>A huge additional maintenance effort would be required to make this work.</li>
+        <li>There are other <a href="https://search.f-droid.org/?q=video+player">apps</a> available that deal with offline playback, in case the phone doesn't have one.</li>
```

Comment:
```suggestion
        <li>There are other <a href="https://search.f-droid.org/?q=video+player">apps</a> available that deal with offline playback, in case the phone doesn't have one.</li>
        <li>One option that could reduce friction between using NewPipe and an external player is [Just Video Player](https://f-droid.org/en/packages/com.brouken.player/), available on F-Droid, which is also based on ExoPlayer like NewPipe's player.</li>
```